lib/puppet/type/iis_application.rb

```diff
@@ -27,7 +37,8 @@ def self.title_patterns
   end
 
   newparam(:applicationname, :namevar => true) do
-    desc "The name of the Application. The virtual path of an application is /<applicationname>"
+    desc "The name of the application. The virtual path of the application is
+          '/<applicationname>'."
```

Comment:
Is this change to accommodate line length?

Reply:
Yes. For a while I was wrapping at only approximately  80 chars, but it got annoying to be in the business of making a hundred judgement calls about when to wrap and when not to wrap, so I switch to a programmatic 80 char wrap.